sdcm/nemesis.py

```diff
@@ -2800,6 +2802,54 @@ def _verify_cdc_feature_status(self, keyspace: str, table: str, cdc_settings: di
             assert actual_cdc_settings == cdc_settings, \
                 f"CDC extension settings are differs. Current: {actual_cdc_settings} expected: {cdc_settings}"
 
+    def _add_new_node_in_new_dc(self) -> BaseNode:
+        new_node = self.cluster.add_nodes(1, dc_idx=0, enable_auto_bootstrap=True)[0]  # add node
+        with new_node.remote_scylla_yaml() as scylla_yml:
+            scylla_yml.rpc_address = new_node.ip_address
+            scylla_yml.seed_provider = [SeedProvider(class_name='org.apache.cassandra.locator.SimpleSeedProvider',
+                                                     parameters=[{"seeds": self.tester.db_cluster.seed_nodes_ips}])]
+        new_node.remoter.run("sudo sh -c 'echo dc_suffix=_nemesis_dc >> /etc/scylla/cassandra-rackdc.properties'")
+        self.cluster.wait_for_init(node_list=[new_node], timeout=300,
+                                   check_node_health=False)  # see _add_and_init_new_cluster_node
+        self.cluster.wait_for_nodes_up_and_normal(nodes=[new_node])
+        return new_node
+
+    def _write_read_data_to_multi_dc_keyspace(self, datacenters: List[str]) -> None:
+        InfoEvent(message='Writing and reading data with new dc').publish()
+        write_cmd = f"cassandra-stress write no-warmup cl=ALL n=1000 \
+        -schema 'keyspace=keyspace_new_dc replication(strategy=NetworkTopologyStrategy,{datacenters[0]}=3,{datacenters[1]}=1) \
+        compression=LZ4Compressor compaction(strategy=SizeTieredCompactionStrategy)' \
+        -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=5 -pop seq=1..1000 -log interval=5"
+        write_thread = self.tester.run_stress_thread(stress_cmd=write_cmd, use_single_loader=True)
+        self.tester.verify_stress_thread(cs_thread_pool=write_thread)
+        read_cmd = f"cassandra-stress read no-warmup cl=ALL n=1000 \
+        -schema 'keyspace=keyspace_new_dc replication(strategy=NetworkTopologyStrategy,{datacenters[0]}=3,{datacenters[1]}=1) \
+        compression=LZ4Compressor' \
+        -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=5 -pop seq=1..1000 -log interval=5"
+        read_thread = self.tester.run_stress_thread(stress_cmd=read_cmd, use_single_loader=True)
+        self.tester.verify_stress_thread(cs_thread_pool=read_thread)
+
+    def disrupt_add_remove_dc(self) -> None:
+        InfoEvent(message='Starting New DC Nemesis').publish()
+        node = self.cluster.nodes[0]
+        new_node = self._add_new_node_in_new_dc()
+        system_keyspaces = ["system_auth", "system_distributed", "system_traces"]
```

Comment:
you might be able to get dynamically all system keyspaces using this function `get_any_ks_cf_list` in `cluster.py`

Reply:
these keyspaces was mentioned in Scylla docs to have modified replication strategies.